client/src/components/ManageAccount/ManageAccount.scss

```diff
@@ -149,7 +148,6 @@
 }
 .change-password-form {
   font-size: 14px;
-  display: flex;
   flex-direction: column;
```

Comment:
Going to remove the `change-password-form` class since it's not doing much anymore. (cleared out the flex styles and moved the font-size up to the parent modal body.

Reply:
Also removing the `password-warning` class